### PR TITLE
Apply recomendations from torbjorn. 

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

File: docs/workforce-management-framework/topics/con-workflow-step.adoc [hunk 1]
```diff
@@ -1,7 +1,9 @@
 [id='{context}-con-workflow-step']
-= Building Your First Workflow Step by Reusing a Template
+= Building Your First Workflow Step
 
-A basic knowledge of Angular.js is required to build a *Workflow Step* in {PROJECT_NAME}.
+Steps are internal part of the {PROJECT_NAME} ecosystem.
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/con-workflow-step.adoc]: ```Steps are a internal part of {PROJECT_NAME}```

File: docs/workforce-management-framework/topics/con-workflow-step.adoc [hunk 2]
```diff
@@ -62,7 +64,24 @@ module.exports = {
 };
 ----
 
-== Development
-When developing a custom _Step Package_, clone the link:https://github.com/feedhenry-raincatcher/step-template.git[step-template] repository and make modifications to the cloned repository or alternatively, use the link:https://www.npmjs.com/package/generator-rcstep[generator-rcstep] npm package to create a _Step_.
+== Creating new step
+
+Step can be created manually from template or by using generator.
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/con-workflow-step.adoc]: A step

File: docs/workforce-management-framework/topics/con-workflow-step.adoc [hunk 3]
```diff
@@ -62,7 +64,24 @@ module.exports = {
...
+
+=== Generator
+
+Generator allows to generate step automatically by answering questions.
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/con-workflow-step.adoc]: The Generator allows you to

**Review by austincunningham — DISMISSED** [review]

File: docs/workforce-management-framework/topics/con-workflow-step.adoc (hunk 1 above)
Comment (wtrocki) [reply docs/workforce-management-framework/topics/con-workflow-step.adoc]: You do not like ecosystem? :) Yeah removing that:)

**Review by austincunningham — APPROVED** [review]

File: docs/workforce-management-framework/topics/con-workflow-step.adoc [hunk 4]
```diff
@@ -1,7 +1,9 @@
 [id='{context}-con-workflow-step']
-= Building Your First Workflow Step by Reusing a Template
+= Building Your First Workflow Step
 
-A basic knowledge of Angular.js is required to build a *Workflow Step* in {PROJECT_NAME}.
+Steps are internal part of the {PROJECT_NAME}.
+Guide bellow describes structure of the step and some basic priciples required build it.
+A basic knowledge of Angular.js is required to build a Workflow Step.
```
Comment (paolobueno) [inline docs/workforce-management-framework/topics/con-workflow-step.adoc]: nitpick: `AngularJS` for proper capitalization.

**Review by paolobueno — APPROVED** [review]

**Pull request merged**